src/main/features/requirements/controller/ca-whereWorkDone.ts

```diff
@@ -7,22 +7,37 @@ import { LoggTracer } from '../../../common/logtracer/tracer';
 
 export const CA_GET_WHERE_WORK_DONE = async (req: express.Request, res: express.Response) => {
   const { SESSION_ID } = req.cookies; //jwt
-  const { projectId, releatedContent, isError, errorText, dimensions } = req.session;
+  const { projectId, releatedContent, isError, errorText, dimensions,currentEvent } = req.session;
```

Comment:
Is this currentEvent declaration really required?

Reply:
yes to get the assessmentID